dev/com.ibm.ws.kernel.boot.core/src/com/ibm/ws/kernel/productinfo/ProductInfo.java

```diff
@@ -225,6 +227,41 @@ public String getProperty(String key) {
         return value;
     }
 
+    /**
+     * Helper method for determining if product is EARLY_ACCESS edition
+     */
+    public boolean isBeta() {
+        boolean isBeta;
+        String productEdition = this.getEdition();
+        if (productEdition == null) {
+            isBeta = false;
+        } else {
+            isBeta = productEdition.equals(EARLY_ACCESS);
+        }
+        return isBeta;
+    }
+
+    /**
+     * Sets the com.ibm.ws.beta.edition JVM property to true if the Liberty version is EARLY_ACCESS,
+     * otherwise sets it to false.
+     */
+    public static void setBetaEditionJVMProperty() {
+        try {
+            if (System.getProperty(BETA_EDITION_JVM_PROPERTY) == null) {
```

Comment:
So what happens if the user has set this system property to "YES"? Please can you add a `public static boolean getBetaEdition()` method for internal components to use to get this value really easily? They shouldn't need to worry about what the exact value of the system property is or wether they need to call the method inside a doPriv or even use `ThreadIdentityManager.runAsServer()`.

Reply:
We should also socialize this at the next EOI, partly to make sure everyone knows that it exists but also to check that it forefills their requirements and that they don't end up writing their own code again.

Reply:
Added the static method getBetaEdition() to ProductInfo in latest commit.